policy/release/collection/slsa3.rego

```diff
@@ -0,0 +1,7 @@
+#
+# METADATA
+# title: slsa3
+# description: |-
+#   Includes policy rules required to meet SLSA Level 3. This is currently a
+#   work in progress and functionally the same as the slsa2 collection.
+package policy.release.slsa3
```

Comment:
I was pondering whether it's worth making an explicit package naming convention for these collection docs. E.g. it might be `policy.release.collection-slsa3` or `policy.release.collection-annotations.slsa3`. Conclusion: the directory is a good enough distinction, and the antora script doesn't require it, so let's keep it simple. What do you think?

Reply:
I wouldn't be against `policy.release.collection.slsa1` as that gives a distinct namespace that matches the file structure. This is minor though. I'm happy either way.

Reply:
Easy to tweak, not sure what would be the best option though... Perhaps what @lcarva notes...